.drone.yml

```diff
@@ -1311,30 +1311,16 @@ steps:
   environment:
     GITHUB_PRIVATE_KEY:
       from_secret: GITHUB_PRIVATE_KEY
-- name: Wait for docker
-  image: docker
-  commands:
-  - timeout 30s /bin/sh -c 'while [ ! -S /var/run/docker.sock ]; do sleep 1; done'
-  volumes:
-  - name: dockersock
-    path: /var/run
-- name: Build artifacts
-  image: docker
+- name: Delegate build to GitHub
+  image: golang:1.18-alpine
```

Comment:
Any particular reason for 1.18?

Reply:
Not particularly. Vaguely matching Go version in tooling `go.mod`, but that's it.